sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -285,6 +285,26 @@ private[sql] object CatalogV2Util {
       case _: NoSuchNamespaceException => None
     }
 
+  def loadTable(
```

Comment:
wondering why we do not choose to modify existing `loadTable(catalog: CatalogPlugin, ident: Identifier)` above?

Reply:
Should modify the existing one. Fixed.